### DISCUSSION TITLE
Two new aliens the Rienli and the Ta'ne

### DISCUSSION THREAD

**Discussion by AcidicPancakes** [endless-sky/endless-sky#9742] [Ideas]
Two fates intertwined

Hello, I have an idea for two factions that I adapted from a scifi novel that I ended up only writing two chapters of before realizing I can't do dialog.

Off both the rim and the dirt belt is an area of space disputed between two species the Rienli a tall reptilian species  and the Ta’ne a small six limbed mammalian species. They both developed in a star system with a unique orbital pattern (based on the orbital pattern of Janus and Epimetheus, two moons of Saturn) where once a year the other planet is fully visible in the skies of these planets. This has profoundly influenced both species with ceremonies and festivals coinciding with the appearance of the other planet in the sky. Anyway once they achieved first contact (around late industrial era) they established a peaceful coexistence. This sadly didn’t last for any significant period of time ending when a major power on Rielia nucked a small Ta'ne nation leading to brief conflict that had no clear victor this lead to a peace treaty that is still in place by the beginning of the game. their space is ether connected to human space through multiple systems containing nothing but dangerous anomalies and one inhabited system that's just a fuel depot (humans will have slight knowledge of the two species the archons/drak/quarg think its enough) or disconnected (if it is too much trouble to introduce another pre hai reveal species). if connected humans will have first contacted them about 50 years before the start and would have some very limited trade (you can only land on one planet each).

Now I guess this is where the game starts with you having the option to visit them. But then things really heat up not long after the free worlds declare independence with the two species starting a war (can happen any time if disconnected) while also sending a message to your government that in exchange for you to help with their war they will help with yours by supplying tech, medicine, food ect. So then you travel to their space and either help the Rienli or the Ta'ne (your preference) this side story can lead to 4 different outcomes depending on your actions. The first one is the rienli victory where the Ta'ne are subjugated, the Ta'ne victory the inverse of the previous victory. Then you have the option to conspire with the peace loving factions that are present in both civilizations you can help overthrow both ruling governments of the rienli and the Ta'ne. This will cause them to form a union with peaceful coexistence for both species or if you fail the coup then both species will become hostile and prevent you from trading or traveling in their space (it shouldn't be too hard to modify the story if it is disconnected).

 so now let's talk about there tech since both species achieved hyperdrive only about 300 years ago there tech is weaker and more bulky than human tech (for example there smallest shield generator takes up about 20 outfit space for only 75% the power) but overall there tech is cheaper than human tech so it may be worth it in the early game. They have some unique outfits that could easily be applied on any ship first is the weapon systems. the weapons present on there ships are the same for both species with the smaller weapon class being the ark/tesla/solar flare these weapons produce an ark of electricity in front of the ship using it they are particularly good at disabling shields and are mounted on small escort crafts and fighters they take up two gun slots and take up a lot of energy (leading to the fuel cells being the only piece of tech more advanced than what you would find in human space). while the big weapons star shot/galaxy shot/universe shot are spinal mounted to their battleships and use physical projectiles that easily decimate the hulls of enemy ships but can be stopped by weak shields and for hand to hand they don't use projectiles and it shows with both options being the worst hand to hand options in the game with the Rienli using spears and the Ta'ne using swords.

for the truly unique outfits the Rienli have the dash drive a semi hyperdrive that causes your ship to move at an extremely fast speed (faster than any afterburner) with the disadvantage of not being able to steer it costs 50 fuel to engage and has an outfit space of 25. the Ta'ne have developed the echo shield that when engaged (50 fuel 45 outfit space) it spawns two identical ships (same design as the ship that activated it)they cannot attack but will absorb damage and cause the enemy to target them more often than you the echo ships will not have shields but will have about 60% of your hulls current health the downside is that it causes your current shields to be much weaker with the max being 50% of your normal shields along with a large energy drain.

I haven't put much thought into the standard outfits but everything aside from the fuel cells will be bigger and weaker that what's in human space. now onto ship designs the rienli favor triangles with most of their ships have a hull made up of multiple triangles while the Ta'ne favor sleek designs think uss voyager from star trek and if you finish their story before fighting the pug they will send a ship to join the pug invasion fleet.

The ideologies of these two species are as diverse as humans with the Rienli being slightly more explorative and the Ta'ne being slightly more religious with the government of both at the start of the game being a monarchy and if they unify they will become democratic.

That is all I will be particularly slow to respond to comments but since these two are from a previous project of mine I can answer most questions asked.

Maps made in microsoft paint back in 2021. 
(https://docs.google.com/document/d/14ZAxR5KZvpj2kJYyOxUKgLVCN1XvuAgkMGoh6s4WFas/edit?usp=sharing)

**Comment by Zitchas** [comment]
Sounds interesting! I'm not sure if it is all fully capable of being implemented in ES as described here, but regardless, sounds rather neat!
